openpype/hosts/blender/api/capture.py

```diff
@@ -6,7 +6,7 @@
 import bpy
 
 from .lib import maintained_time
-from .plugin import deselect_all, create_blender_context
+from .plugin import deselect_all
```

Comment:
`create_blender_context` is still referenced twice in this file, you shouldn't remove its import.

Reply:
Actually... It's not in `plugin.py`, but its references in `catpure.py` are problematic then

Reply:
It doesn't exist in the `plugin.py` and it is never called with the current implementation. It is removed in later code. It is not clean I agree.